src/lib/cache.coffee

```diff
@@ -66,17 +66,16 @@ module.exports = (dnschain) ->
                     @cache.setex(key, ttl, JSON.stringify value) if not err2
                     valueDoer err, key, value
 
-        resolveBlockchain: (resolver, path, options, cb) ->
+        resolveResource: (requestFn, serialization, cb) ->
             if @blockchainEnabled and resolver.cacheTTL?
```

Comment:
Oops, looks like I screwed up the build with bad advice.... darn. Forgot that we need `resolver`.